homeassistant/components/vera/config_flow.py

```diff
@@ -0,0 +1,13 @@
+"""Config flow for Vera."""
+from homeassistant import config_entries
+
+from .const import CONF_CONTROLLER, DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class VeraFlowHandler(config_entries.ConfigFlow):
```

Comment:
Pass the domain name as domain keyword argument and remove the decorator.

https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#defining-your-config-flow